osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
Nope. This is already implemented:
determining the absolute position of a sensor requires two transformations. 
See SensorData::mounting_position and Vehicle::relative_ego_origin